### PR TITLE
Adding new patternfly-react core

### PR DESCRIPTION
**Pull request by ryelo**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,18 +3,19 @@
   "version": "0.1.0",
   "private": false,
   "dependencies": {
+    "@patternfly/patternfly-next": "^1.0.3",
```
Comment (dmiller9911) [inline package.json]: This is only needed if you use things outside of what react-core has.

File: package.json [hunk 2]
```diff
@@ -3,18 +3,19 @@
...
+    "@patternfly/react-core": "0.0.4",
+    "@patternfly/react-tokens": "^1.0.0",
```
Comment (dmiller9911) [inline package.json]: Same with this.  If you don't plan to reference the variables from pf4 inside of JS this is not needed.

Comment (ryelo) [reply package.json]: Sweet. Thanks @dmiller9911

File: src/_patternfly_specific.scss [hunk 3]
```diff
@@ -1,9 +1,9 @@
...
-  @import '~patternfly-react/dist/sass/_patternfly-react.scss';
-}
\ No newline at end of file
+// .special-patternfly {
+//   $font-path: '~patternfly/dist/fonts/';
+//   $img-path: '~patternfly/dist/img/';
+//   $icon-font-path: '~patternfly/dist/fonts/';
+//   @import '~patternfly/dist/sass/_patternfly.scss';
+//   @import '~patternfly-react/dist/sass/_patternfly-react.scss';
+// }
```
Comment (Kinlaw) [inline src/_patternfly_specific.scss]: Any reason not to just delete this?

Comment (ryelo) [reply src/_patternfly_specific.scss]: Deleted!

**Comment by Kinlaw** [comment]
@ryelo looks good. Can you rebuild package-lock.json?

**Pull request merged**